src/app/new-page/component/content-accordion/content-accordion.component.css

```diff
@@ -7,59 +15,57 @@
   width: 100%;
   text-align: left;
   font-size: 16px;
+  font-family:
+    Source Sans Pro,
+    sans-serif !important;
   border: none;
   outline: none;
   transition: 0.4s;
-  color: var(--kg-white);
-  background-color: var(--kg-blue);
-  transition: all 0.3s ease 0s;
+  color: var(--kg-gray-dark);
+  background-color: var(--kg-gray-light);
+  transition: all 0.3s linear;
   position: relative;
+  display: flex;
+  align-items: center;
 }
 
 .accordion-header:hover {
-  background-color: var(--kg-blue-light);
+  background-color: var(--kg-gray);
+  color: var(--kg-gray-dark);
+}
+.accordion-header:active {
+  background-color: var(--kg-gray-medium-dark);
 }
 
 .accordion-header i {
   position: absolute;
   right: 20px;
-  transform: translateY(-50%);
-  transition: all 0.3s ease 0s;
-  top: 50%;
+  -moz-transition: all 0.2s linear;
+  -webkit-transition: all 0.2s linear;
+  transition: all 0.2s linear;
 }
 
-.accordion-header i.fa-plus {
-    opacity: 1;
-}
-.accordion-header i.fa-minus {
-  opacity: 0;
-}
-
-.accordian.mw568.active .accordion-header i.fa-plus  {
-  opacity: 0;
-}
-.accordian.mw568.active .accordion-header i.fa-minus  {
-  opacity: 1;
+.accordion.mw568.active .accordion-header i.fa-chevron-down {
+  -ms-transform: rotate(180deg);
```

Comment:
You no longer need to prefix `transform` as 98% of browsers support it.

https://caniuse.com/?search=transform